build-logic/publishing/src/main/kotlin/build-logic.java-published-library.gradle.kts

```diff
@@ -17,6 +17,8 @@ java {
 publishing {
     publications {
         create<MavenPublication>(project.name) {
+            artifactId = "jdbc-yugabytedb"
```

Comment:
Why does the pgjdbc upstream code does not need to specify the artifactId and version here but we need to?

Reply:
I had ported all the changes from previous master that were missing.
But now if I remove only this change, I'm still able to publish the jar. May be we can get rid of this then.

Reply:
Reverted this change